### ISSUE TITLE
Metrics parameters argument to RiemannianMetric

### ISSUE THREAD

**Issue by ddundo** [pyroteus/animate#89]
How about we pass metric parameters as an argument to `RiemannianMetric` instead of using `set_parameters`? We could then introduce the `update_parameters` method.

**Comment by jwallwork23** [comment]
Yeah it would be simple enough to have `__init__` accept a `metric_parameters` kwarg and call `set_parameters`/`update_parameters` inside. PR welcome!
